.github/workflows/dev-image-update.yaml

```diff
@@ -95,12 +98,21 @@ jobs:
     needs: update_image
     uses: ./.github/workflows/helmtagbump.yaml
     secrets: inherit
+  cherry_pick_image_to_production_gcr:
+    needs: update_image
+    uses: DataBiosphere/jade-data-repo/.github/workflows/cherry-pick-image.yaml@1.536.0
```

Comment:
Food for thought: would we want this to switch to an `@latest` version to avoid needing to have to update if we ever make a change to the cherry-pick-image action?

Reply:
This follows the convention that we use with the datarepo-actions repo. While, yes, it requires an extra PR to update the version, it also allows us to make breaking changes to the jade-data-repo action without immediately breaking the ui action (We can merge an API change, then merge the UI change with no downtime). It is also slightly more secure (Although, given that we control both repos, this really shouldn't be an issue).

Reply:
yeah, that's totally fair